client/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+    <div id="app">Hello Dude</div>
```

Comment:
I think this script source needs to be set to your webpack bundle instead but it sounds like you're getting to that in the next feature branch.